docs/conf.py

```diff
@@ -105,6 +105,9 @@ def _get_project_meta():
 #
 html_theme = "pydata_sphinx_theme"
 
+html_logo = "_static/logo.png"
+html_favicon = "_static/favicon.ico"
```

Comment:
Not sure where and how this is used (?)

Reply:
It should appear on the docs top left and the favicon is the icon that shows on website tabs.

Reply:
<img width="1508" alt="image" src="https://github.com/user-attachments/assets/c0cdb021-ed92-4aa3-8d33-9ddcf40b74da" />

Reply:
You are right, now it shows. Last time I did not see it, maybe browser cache or so.